easybuild/easyconfigs/s/SciPy-bundle/scipy-1.10.1_fix-lobpcg-test.patch

```diff
@@ -0,0 +1,185 @@
+see https://github.com/scipy/scipy/issues/17954 + https://github.com/scipy/scipy/pull/17975
```

Comment:
Can we use the diff instead of the patch? The patch is more complicated as it adds and reverts some changes, but the diff is fairly simple. See https://patch-diff.githubusercontent.com/raw/scipy/scipy/pull/17975.diff

Reply:
Neither the .patch or .diff from that PR apply.

See https://github.com/boegel/easybuild-easyconfigs/pull/91 for a modified version that does.